napari/plugins/_npe2.py

```diff
@@ -374,7 +386,7 @@ def _npe2_manifest_to_actions(
             enablement=cmd.enablement,
             callback=cmd.python_name or '',
             menus=cmds.get(cmd.id),
-            keybindings=[],
+            keybindings=key_bindings.get(cmd.id),
```

Comment:
```suggestion
            keybindings=key_bindings.get(cmd.id, []),
```
If I understand correctly, we would like to provide an empty list, not None?

Reply:
according to the spec of an action, [`keybindings` is an optional field](https://github.com/pyapp-kit/app-model/blob/0c9e6ac250eb7b39b797bea1d33bfc2d4c8f65e3/src/app_model/types/_action.py#L48), meaning that we can provide `None` to denote that the plugin left this field blank